src/sentry/utils/sdk.py

```diff
@@ -236,6 +237,12 @@ def before_send(event: Event, _: Hint) -> Event | None:
             event["tags"]["silo_mode"] = str(settings.SILO_MODE)
         if settings.SENTRY_REGION:
             event["tags"]["sentry_region"] = settings.SENTRY_REGION
+
+    if "contexts" in event:
+        event["contexts"]["flags"] = get_flags_serialized()
+    else:
+        event["contexts"] = {"flags": get_flags_serialized()}
```

Comment:
logaf l: idk if this is more pythonic or not but I prefer to init `event["contexts"]` to a dict first and then be able to updates `flags` in the same fashion aftwards